src/metadata.ts

```diff
@@ -24,6 +24,8 @@ export interface AppMetadata extends Metadata {
    */
   readonly description: string
 
+  readonly main?: string
```

Comment:
If I understood correctly interface AppMetadata holds all "useful" properties of the application package.json for the sake ot typesafety. I thought I only gave access to main (an already existent option). Please correct me if I am wrong.